dokka-runners/dokka-gradle-plugin/src/testFunctional/resources/KotlinDslAccessorsTest/root-project-v2-migration-helpers.txt

```diff
@@ -493,61 +493,6 @@ fun DependencyHandler.`dokkaHtmlRuntime`(
 fun DependencyHandler.`dokkaHtmlRuntime`(dependencyNotation: Any): Dependency? =
     add("dokkaHtmlRuntime", dependencyNotation)
 
-@Deprecated(message = "The dokkaJavadocPartialPlugin configuration has been deprecated for dependency declaration. Please use the 'dokkaJavadocPlugin' configuration instead.")
```

Comment:
Why those files were changed? Those changes are not present also in https://github.com/Kotlin/dokka/pull/3801